dev/integration_tests/abstract_method_smoke_test/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.abstract_method_smoke_test">
-
+    <uses-permission android:name="android.permission.WAKE_LOCK"/>
```

Comment:
Add a TODO to remove this or figure out why it's needed, with a link to a bug for it.